hana/saptune.sls

```diff
@@ -11,4 +11,8 @@ apply_saptune_solution_{{ host }}_{{ name }}:
   saptune.solution_applied:
     - name: {{ saptune_solution }}
 
+start_saptune_daemon_{{ host }}_{{ name }}:
```

Comment:
I would like maybe here that we add a comment/rationale why we are starting the daemon.

E.g I would add here:

```
# start saptune daemon will ensure that after a reboot of the host, the solution applied via saptune is still compliant

```